pkg/setting/setting.go

```diff
@@ -1691,6 +1692,17 @@ func readUserSettings(iniFile *ini.File, cfg *Cfg) error {
 		return errors.New("the minimum supported value for the `user_invite_max_lifetime_duration` configuration is 15m (15 minutes)")
 	}
 
+	userLastSeenUpdateIntervalVal := valueAsString(users, "last_seen_update_interval", "15m")
+	userLastSeenUpdateIntervalDuration, err := gtime.ParseDuration(userLastSeenUpdateIntervalVal)
+	if err != nil {
+		return err
+	}
+
+	cfg.UserLastSeenUpdateInterval = userLastSeenUpdateIntervalDuration
+	if cfg.UserInviteMaxLifetime < time.Minute*5 {
+		return errors.New("the minimum supported value for the `last_seen_update_interval` configuration is 5m (5 minutes)")
```

Comment:
Instead of returning an error here we should emit an warning log and set the minimum value